pkg/driver/node.go

```diff
@@ -89,6 +89,13 @@ func newNodeService(driverOptions *DriverOptions) nodeService {
 		panic(err)
 	}
 
+	if driverOptions.startupTaintRemoval {
+		err := cloud.RemoveNodeTaint(cloud.DefaultKubernetesAPIClient, AgentNotReadyNodeTaintKey)
+		if err != nil {
+			klog.InfoS("Node agent-not-ready taint error", "error", err)
```

Comment:
What if someone enables `startupTaintRemoval` even when running on other COs? This will cause the node service to fail to run.